src/components/button/AposButton.vue

```diff
@@ -124,6 +126,47 @@ export default {
     background-color: transparent;
   }
 
+  .apos-button--gradient-on-hover {
+    &:after {
+      z-index: 1;
+      content: '';
+      position: absolute;
+      top: 0;
+      left: 0;
+      width: 100%;
+      height: 100%;
+      background-image: linear-gradient(46deg, #CC9300 0%, #EA433A 26%, #B327BF 47%, #6666FF 76%, #00BF9A 100%);
+      opacity: 0;
+      transition: opacity 0.3s ease;
+    }
+    &:hover:after,
+    &[disabled].apos-button--busy:after {
+      opacity: 1;
+    }
+    // extra strength :sweat-smile:
+    &.apos-button[disabled].apos-button--busy {
+      border: none;
+    }
+    &[disabled].apos-button--busy:after {
+      animation: animateGradient 5s ease-in-out infinite;
+    }
+    .apos-button__label {
+      position: relative;
+    }
+    &[disabled].apos-button--busy .apos-loading__svg {
+      color: var(--a-white)
+    }
+    .apos-button__label,
+    .apos-loading {
+      z-index: 2;
```

Comment:
If we're using multiple z-index values here, it would be smart to establish the button as a stacking context by adding `z-index: 0` to it.

Reply:
absolutely roiled by this!

Reply:
but also resolved